CMakeLists.txt

```diff
@@ -1,4 +1,4 @@
-cmake_minimum_required(VERSION 3.10) # CMP0083 NEW
+cmake_minimum_required(VERSION 3.8) # CMP0083 NEW

```

Comment:
@AJenbo are you fine with this? perhaps 3.10 is required for the LTO check? Otherwise it seems to work on 3.8

Reply:
Ubuntu 18.04, Android Stuido, Visual Stuido and possibly Xcode are all stuck on 3.10 so that is our upper limit. CheckIPOSupported appears to have been introduced in 3.9 so I guess that is our lower limit. This also explains why @Cowcat5150 was having issues with LTO detection if he is running CMake 3.8. @Cowcat5150 is it possibl for you to upgrade to at least 3.9?

Reply:
MorphOS SDK seems to be stuck at 3.8. On crosstools this shouldn't be an issue though. But my suggestion to put Platform defines above LTO check could be a workaround on systems where this is not supported. (as stated in another comment below)

Reply:
I cannot upgrade to upper cmake till a newer warpos SDK is done against newer MOS SDK that has 3.10 cmake.

Reply:
Download the mos SDK and compile cmake from there yourself? Shouldn’t be too hard.

Reply:
I could try to just copy newer cmake package and see what happens, but installing newer SDK over the old and being ok with it thinking that nothing is gonna happen with the warpos side is like making christmas cakes with cocaine instead of sugar.

Reply:
And what the hell is "dismissed review" here? I don't dismiss anyone. :)

Reply:
Crap, cmake from newer MOS still 3.8.....

Reply:
Setting if(NOT WARPOS) on top Cmakelists for choosing 3.10 or 3.8 cmake is useless.